CS4850Group5A/VirtuDoc
# #181 Landing page POC

# Summary
- Added landing page proof-of-concept.
- Changed `HomeController` to render `landing.html`.
- Removed `index.html` placeholder page.

Closes #181

# How to Test
- As per the instructions in the wiki, open the screenshots folder for the GHA run and confirm that the page renders correctly.
- Run the web app locally and confirm that the landing page shows up when you navigate to `/`.

# Comments
Currently looking for something better to use as the text in the center of the page. Other suggested changes to the design of the page are also welcome.
